### PR TITLE
#160709740 Update food-option

### PR DESCRIPTION
**Pull request by adex001**
#### What does this PR do?
**1.** Updates a food item in the menu.

#### Description of Task to be completed?
N/A
#### How should this be manually tested?
Clone the repository, 'npm i' to install all dependencies and 'npm start', then open Postman and navigate the endpoints.
#### What are the relevant pivotal tracker stories?
#[160709740](https://www.pivotaltracker.com/story/show/160709740)

### CODE REVIEW

File: server/controller/menu.js [hunk 1]
```diff
@@ -54,6 +54,34 @@ class MenuController {
...
+  * UpdateFoodItem Controller updates a meal in the database
+  * @param {string} req - The request to the server
+  * @param {string} res - The response from the server.
+  */
+  static updateFoodItem(req, res) {
+    const { mealId } = parseInt(req.params, 10);
+    const {
+      mealName, mealImageUrl, mealDescription, mealPrice,
+    } = req.body;
+    const updateMealQuery = `UPDATE menu SET mealName = '${mealName}', mealImageUrl = '${mealImageUrl}', mealDescription = '${mealDescription}', mealPrice = '${mealPrice}' WHERE mealId = ${mealId} RETURNING *;`;
```
Comment (KaiserPhemi) [inline server/controller/menu.js]: #### Critical
Format this code to be more readable.

**Review by KaiserPhemi — CHANGES_REQUESTED** [review]
Implement the feedback and you're good to go.

File: server/controller/menu.js (hunk 1 above)
Comment (adex001) [reply server/controller/menu.js]: I have implemented this feedback

Comment (KaiserPhemi) [reply server/controller/menu.js]: #### Nitpick
Put each of those variables on a new line since you're using template literals.

**Review by KaiserPhemi — APPROVED** [review]

**Pull request merged**